packages/auto-install/LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2018 Rich Harris
```

Comment:
you can remove this file. the LICENSE from the root gets copied into the package for publishing